gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-tcp-proxy/src/main/resources/schemas/sharedConfiguration/schema-form.json

```diff
@@ -26,6 +26,32 @@
           "minimum": 0,
           "title": "Reconnect interval",
           "description": "Interval in ms between retries connecting to the target"
+        },
+        "idleTimeout": {
+          "type": "integer",
+          "default": 0,
+          "minimum": 0,
+          "title": "Idle timeout (ms)",
+          "gioConfig": {
+            "banner": {
+              "title": "Idle timeout",
+              "text": "Maximum time a connection will stay in the pool without being used in milliseconds. Once the timeout has elapsed, the unused connection will be closed, allowing to free the associated resources. Zero means no timeout."
```

Comment:
There is no concept of pooling here.

"Maximum time a TCP connection will stay active if no data is received or sent in milliseconds. Once the timeout has elapsed, the unused connection will be closed, allowing to free the associated resources. Zero means no timeout."